1436790/1438462: Part 34: 20 files with MSDN link fixes.

DennisLee-DennisLee opened the pull request:
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

DennisLee-DennisLee committed:
20 more files with MSDN link fixes.

DennisLee-DennisLee committed:
Merge branch 'master' of https://github.com/dotnet/docs into v-dele-1438462-034

mairaw commented on docs/framework/wcf/service-trace-viewer-tool-svctraceviewer-exe.md:
```diff
@@ -446,4 +446,4 @@ Windows Communication Foundation (WCF) Service Trace Viewer Tool helps you analy
 - [Using Service Trace Viewer for Viewing Correlated Traces and Troubleshooting](../../../docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md)
 - [Configuring Tracing](../../../docs/framework/wcf/diagnostics/tracing/configuring-tracing.md)
-- [Activity Tracing and Propagation for End-To-End Trace Correlation](https://msdn.microsoft.com/library/2c11a905-64f8-47b5-bae5-a74fc666137e)
+- [Activity Tracing and Propagation for End-To-End Trace Correlation](diagnostics/tracing/end-to-end-tracing.md)
```
```suggestion
- [End-to-End Tracing](./diagnostics/tracing/end-to-end-tracing.md)
```

mairaw commented on docs/framework/wcf/using-sessions.md:
```diff
@@ -131,7 +131,7 @@ In Windows Communication Foundation (WCF) applications, a *session* correlates a
   
 ### Sharing InstanceContext Objects  
- You can also control which session-based channel or call is associated with which <xref:System.ServiceModel.InstanceContext> object by performing that association yourself. For a complete example, see [InstanceContextSharing](https://msdn.microsoft.com/library/4a6a46d7-b7d7-4bb5-a0dd-03ffa3cbc230).  
+ You can also control which session-based channel or call is associated with which <xref:System.ServiceModel.InstanceContext> object by performing that association yourself. For a complete example, see [InstanceContextSharing](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.5/aa354514(v=vs.90)).  
```
```suggestion
 You can also control which session-based channel or call is associated with which <xref:System.ServiceModel.InstanceContext> object by performing that association yourself. 
```

mairaw commented on docs/framework/wcf/wcf-troubleshooting-quickstart.md:
```diff
@@ -96,7 +96,7 @@ public class MyServiceHost : ServiceHost
 -   You might need to open the port to the application. For details, see [Firewall Instructions](../../../docs/framework/wcf/samples/firewall-instructions.md) from the SDK samples.  
   
--   For other possible issues, see the samples topic [Running the Samples in a Workgroup and Across Machines](https://msdn.microsoft.com/library/a451a525-e7ce-452d-9da9-620221260113).  
+-   For other possible issues, see the samples topic [Running the Windows Communication Foundation Samples](samples/running-the-samples.md).  
```
```suggestion
-   For other possible issues, see the samples topic [Running the Windows Communication Foundation Samples](./samples/running-the-samples.md).  
```

mairaw commented on docs/framework/whats-new/index.md:
```diff
@@ -1163,7 +1163,7 @@ SqlClient now automatically provides faster connections to an AlwaysOn Availabil
 **Samples**
 
- There are a number of [WPF Samples](https://msdn.microsoft.com/library/ms771633.aspx) on MSDN. More than 200 of the most popular samples (based on their usage) will be moved into an [Open Source GitHub repository](https://github.com/Microsoft/WPF-Samples). Help us improve our samples by sending us a pull-request or opening a [GitHub issue](https://github.com/Microsoft/WPF-Samples/issues).
+ There are a number of WPF samples on our [Open Source GitHub repository](https://github.com/Microsoft/WPF-Samples). Help us improve our samples by sending us a pull-request or opening a [GitHub issue](https://github.com/Microsoft/WPF-Samples/issues).
```
```suggestion
 There are a number of WPF samples on the [Microsoft/WPF-Samples](https://github.com/Microsoft/WPF-Samples) GitHub repository. Help us improve our samples by sending us a pull-request or opening a [GitHub issue](https://github.com/Microsoft/WPF-Samples/issues).
```

mairaw commented on docs/framework/whats-new/obsolete-types.md:
```diff
@@ -624,7 +624,6 @@ ms.author: "ronpet"
 
 <a name="VBCompat"></a>
 ### Assembly: Microsoft.VisualBasic.Compatibility.dll
-  For information about migrating from Visual Basic 6, see [Visual Basic 6.0 Resource Center](https://msdn.microsoft.com/library/windows/desktop/ms788229).
```
I told you to get rid of VB6 links, but in this case here because it's about VB6 compat, I'd just update to previous-versions. Sorry!

mairaw commented on docs/framework/windows-workflow-foundation/performance.md:
```diff
@@ -418,7 +418,7 @@ public class Workflow1 : Activity
  While WF4 does not have a SQL tracking provider, AppFabric does.  AppFabric’s SQL tracking approach is to subscribe to ETW events with a Windows Service that batches the events and writes them to a SQL table designed for quick inserts.  A separate job drains the data from this table and reforms it into reporting tables that can be viewed on the AppFabric dashboard.  This means that a batch of tr ...
 
- ETW events can be recorded with tools such as logman or xperf.  The compact ETL file can be viewed with a tool like xperfview or converted to a more readable format, such as XML, with tracerpt.  In WF3, the only option for getting tracking events without a SQL database is to create a custom tracking service. For more information about ETW, see [WCF Services and Event Tracing for Windows](../../. ...
+ ETW events can be recorded with tools such as logman or xperf.  The compact ETL file can be viewed with a tool like xperfview or converted to a more readable format, such as XML, with tracerpt.  In WF3, the only option for getting tracking events without a SQL database is to create a custom tracking service. For more information about ETW, see [WCF Services and Event Tracing for Windows](../../. ...
```
```suggestion
 ETW events can be recorded with tools such as logman or xperf.  The compact ETL file can be viewed with a tool like xperfview or converted to a more readable format, such as XML, with tracerpt.  In WF3, the only option for getting tracking events without a SQL database is to create a custom tracking service. For more information about ETW, see [WCF Services and Event Tracing for Windows](../../../docs/framework/wcf/samples/wcf-services-and-event-tracing-for-windows.md) and [Event Tracing - Windows applications](/windows/desktop/etw/event-tracing-portal).
```

mairaw commented on docs/framework/windows-workflow-foundation/samples/corporate-purchase-process.md:
```diff
@@ -102,7 +102,7 @@ This sample shows how to create a very basic Request for Proposals (RFP) based p
 |VendorProposal|A proposal submitted by a vendor to a concrete RFP.|  
 |VendorRepository|The repository of Vendors. This implementation contains an in-memory collection of instances of Vendor and methods for exposing those instances.|  
-|RfpRepository|The repository of Requests for Proposals. This implementation contains uses Linq to XML to query the XML file of Requests for Proposal generated by the schematized persistence. This class implements [System.Runtime.Persistence.IDataViewMapper](https://msdn.microsoft.com/library/system.runtime.persistence.idataviewmapper(v=vs.110).aspx).|  
+|RfpRepository|The repository of Requests for Proposals. This implementation contains uses Linq to XML to query the XML file of Requests for Proposal generated by the schematized persistence. This class implements [System.Runtime.Persistence.IDataViewMapper](https://docs.microsoft.com/previous-versions/dd466483(v=vs.100)).|  
```
I checked the sample and this is not accurate, so we can remove it.

```suggestion
|RfpRepository|The repository of Requests for Proposals. This implementation contains uses Linq to XML to query the XML file of Requests for Proposal generated by the schematized persistence. |  
```

mairaw reviewed (changes requested):
Looks great @DennisLee-DennisLee! A few comments before we merge.

mairaw committed:
Apply suggestions from code review

Accepting all 6 of Maira's autocommittable suggestions. Will get to the manual suggestion later.

Co-Authored-By: DennisLee-DennisLee <v-dele@microsoft.com>

DennisLee-DennisLee committed:
Restored and fixed a VB6 resource center link.

mairaw reviewed (approved):
LGTM. Thanks @DennisLee-DennisLee!
